content/en/docs/reference/kubectl/overview.md

```diff
@@ -71,6 +71,20 @@ Flags that you specify from the command line override default values and any cor
 
 If you need help, run `kubectl help` from the terminal window.
 
+## In-Cluster Auth, Namespace Defaulting
+
+If you are running `kubectl` within a pod, and thus in a cluster, the cli attempts to scope CLI commands to the pod's namespace. 
```

Comment:
Something like this.

```suggestion
By default `kubectl` will first determine if it is running within a pod, and thus in a cluster. It starts by checking for the `KUBERNETES_SERVICE_HOST` and `KUBERNETES_SERVICE_PORT` environment variables and the existence of a service account token file at `/var/run/secrets/kubernetes.io/serviceaccount/token`. If all three are found in-cluster authentication is assumed.

To maintain backwards compatibility, if the `POD_NAMESPACE` environment variable is set during in-cluster authentication it will override the default namespace from the from the service account token. Any manifests or tools relying on namespace defaulting will be affected by this.
```